ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -67,6 +77,11 @@ cat <<EOF >>"${SKIP_TESTS_FILE}"
 # TESTNAME
 performance "Check RPS Mask is applied to atleast one single rx queue on all veth interface"
 
+# SKIPTEST
+# bz### https://issues.redhat.com/browse/OCPBUGS-11750
+# TESTNAME
+tuningcni "sysctl allowlist update should start a pod with custom sysctl only after adding sysctl to allowlist"
```

Comment:
And this will skip it only in 13 version, see function names:  `function create_tests_temp_skip_list_13`

Reply:
I understand, but the bug URL is different. There are 2 bugs, 1 for 4.14 and 1 for 4.13, that's why I defined it that way.